co-log-core/src/Colog/Core/Action.hs

```diff
@@ -239,3 +258,20 @@ infixr 1 <<=
 (<<=) :: Semigroup msg => (LogAction m msg -> m ()) -> LogAction m msg -> LogAction m msg
```

Comment:
Could you please move `Comonadic` combinators at the end of the module? To make this module body more consistent with reexport list.